### PR TITLE
[WIP] do not merge - just for discussion

### PR DESCRIPTION
**Pull request by grzanka**
@marwin1991 @adudzik please take a look to the discussion

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.7.2)
```
Comment (grzanka) [inline CMakeLists.txt]: Make should be added in separate PR fixing issue #21 
documentation should be also modified accordingly

Comment (grzanka) [reply CMakeLists.txt]: There was already another attempt in other branch https://github.com/libamtrack/library/blob/cmake_support/CMakeLists.txt

File: compile_to_js.sh [hunk 2]
```diff
@@ -0,0 +1,475 @@
+#!/bin/bash
+
```
Comment (grzanka) [inline compile_to_js.sh]: This one should go into `Distributions/JavaScript` directory

File: js/parse_headers.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        file.write("\nfuns+=']'\n")
+        write_outro(file)
+
+
+def print_missing_files(directory):
+    print("Files not parsed: ")
+
+    for file in os.listdir(directory):
+        filename = os.fsdecode(file)
+
```
Comment (grzanka) [inline js/parse_headers.py]: This should also go to `Distributions\JavaScript`

File: src/AT_CernlibFuns.c [hunk 4]
```diff
@@ -0,0 +1,24 @@
+//
+// Created by osboxes on 04.09.18.
+//
+#include "AT_CernlibFuns.h"
+
+double CL_denlan(double lambda_landau){
+    return 0;
```
Comment (grzanka) [inline src/AT_CernlibFuns.c]: Why body of these functions is empty ?

File: CMakeLists.txt (hunk 1 above)
Comment (marwin1991) [reply CMakeLists.txt]: This CmakeLists.txt for now is just for JavaScript because in configure you link to FORTRAN and that is why compile to JS fails. I dont get all libamtrack dependecies so can i get some help from you?

File: compile_to_js.sh (hunk 2 above)
Comment (marwin1991) [reply compile_to_js.sh]: done

File: js/parse_headers.py (hunk 3 above)
Comment (marwin1991) [reply js/parse_headers.py]: done

File: CMakeLists.txt (hunk 1 above)
Comment (grzanka) [reply CMakeLists.txt]: Sure - I was planning to make my own CMake settings for libamtrack library. First step will be to deal with Fortran dependencies (i.e. replacing them with C version).
What is required from the side of emscripten ?

Comment (marwin1991) [reply CMakeLists.txt]: The best would be use just GSL for  Landau
https://www.gnu.org/software/gsl/manual/html_node/The-Landau-Distribution.html
And then we can merge it to this branch and see if compile 
then i will try to compile root library to JS, because if it also contains some other language dependencie it will be point less. In the worst case scenerio we can make some "if" in Cmake to when compile to JS. What do you think?

Comment (grzanka) [reply CMakeLists.txt]: Moved to https://github.com/libamtrack/library/pull/30

**Comment by grzanka** [comment]
Obsolete by https://github.com/libamtrack/library/pull/39
